### PR TITLE
Improve display of logged trades

### LINKED ISSUE #486: Unnecessary spacing on reference rows
[This bit](https://i.imgur.com/qOvA27a.png) looks weird and it wastes space, which is important on mobile browsers because there's not a lot of space to begin with.

I was able to [fix this](https://i.gyazo.com/d552d0a46041ff48825de5f3e8ab29fb.png) with css by getting rid of `table-layout: fixed;` and adding `width: 50px;` to the element containing the numbers. However, getting rid of `table-layout: fixed;` has some undesirable side-effects such as [changing the width of other columns if a word is too long](https://i.gyazo.com/11436a131d72dcff1b2c4260733b2a34.png). (Note: while it's unlikely that anyone would need a word as long as the one in the example, the length where stuff starts to break is much smaller on smaller devices.)

Anyway, we should figure out a way to get rid of the unnecessary spacing without breaking other stuff.

### PR DESCRIPTION
**Pull request by Kirzi**
Fixes #486

### CODE REVIEW

**Comment by YaManicKill** [comment]
You got a screenshot? Can't be bothered running it to see what it looks like.

**Comment by Kirzi** [comment]
https://imgur.com/a/ZRZzk

File: assets/views/home/references.ejs [hunk 1]
```diff
@@ -1,5 +1,15 @@
 <div class="row">
   <table class="table" id="referencesTable">
+    <thead>
+      <tr class="no-space">
+        <th class="col-xs-1 flair-number"></th>
```
Comment (YaManicKill) [inline assets/views/home/references.ejs]: This seems like a weird way to force a width on a column, adding extra elements.

Could we not do

```
tr td:first-child {
  width: 50px;
}
```

Which should fix the width on each of the first cells on each row? Might be nicer.

File: assets/styles/importer.less [hunk 2]
```diff
@@ -217,4 +216,24 @@ td h3 {
 
 .optional {
   font-weight: normal;
-}
\ No newline at end of file
+}
+
+.ban-message {
```
Comment (YaManicKill) [inline assets/styles/importer.less]: Why did you move these?

Comment (Kirzi) [reply assets/styles/importer.less]: Figured it'd be nicer to have all the "misc"  CSS all in one place.

File: assets/views/home/references.ejs (hunk 1 above)
Comment (Kirzi) [reply assets/views/home/references.ejs]: Tried that. It doesn't work.



Anything else I tried would break something else.

Comment (YaManicKill) [reply assets/views/home/references.ejs]: Oh yes, because that would do it for the titles as well. 

Just to humour me, try

```
tr td:first-child:not(.firstHeading) {
  width: 1000px;
}
```

If it doesn't work, that's fine, I'll approve this.

Comment (YaManicKill) [reply assets/views/home/references.ejs]: Ah wait no, of course that won't work, because it only applies to the first. Right, fine, have your approval, but only because this html is already a crapshow.
